Resources/Private/Javascript/Components/UpdateQuantity.js

```diff
@@ -5,22 +5,10 @@ const updateQuantity = ({ proxy, lineItemId }) => {
   const apiClient = useApiClient();
 
   const update = debounce(async (value) => {
-    const result = await apiClient.updateLineItem(
+    apiClient.updateLineItem(
```

Comment:
and again, feedback is important

Reply:
I've added an utility class while loading